site.yml

```diff
@@ -159,6 +159,30 @@
       tags: ['openstack', 'nova', 'data']
       when: not ironic.enabled|bool
 
+- name: ceph monitors
+  gather_facts: force
+  hosts: ceph_monitors
+  roles:
+    - role: ceph-common
+      tags: ['ceph', 'ceph-common']
+      when: ceph.enabled|bool
```

Comment:
Why isn't ceph-common just made a role dependency of ceph-monitor?

Ditto for ceph-osds? If one role won't work without the other than that should be expressed as a dependency rather than an assumption that the developer knows to add both.